vscode-lean4/src/rpc.ts

```diff
@@ -18,7 +18,7 @@ export class Rpc {
                 try {
                     this.sendMessage({ seqNum, result: await this.methods[name](...args) })
                 } catch (ex) {
-                    this.sendMessage({ seqNum, exception: ex === undefined ? 'error' : ex })
+                    this.sendMessage({ seqNum, exception: ex === undefined ? 'error' : ex.toString() })
```

Comment:
Is there a reason for this change?  I'm not sure what it fixes, but it breaks exceptions like `{code: 3278}` which are now rendered as `[object Object]`.

Reply:
Yes, the extension-side was receiving exceptions with only closure members in the prototype (like a custom `ex.toString`) which couldn't be JSON-serialized and the infoview saw an empty object. But I see that there are other cases in which we do not want to call that. I'll add some cases.

Reply:
I can't find the case with closures anymore, reverting this change.